cypress/support/pages/news/pageCommonNews.js

```diff
@@ -3,6 +3,67 @@
 class News_Common {
 
   static #newsOverviewNavigationButton = '[data-testid="Neuigkeiten"]'
+  static #createNewNews = '[data-testid="create-news-btn"]'
+  static #newsTitleInput = '[data-testid="news_title"]'
+  static #newsDescription= '[contenteditable="true"]'
+  static #newsDateInput= '[data-testid="news_date"]'
+  static #newsTimeInput= '[data-testid="news_time"]'
+  static #newsCreateButton= '[data-testid="btn_news_submit"]'
+  static #newsTitle = '[id="page-title"]'
+  static #newsDescriptionVisibe = '[class="ckcontent"]'
```

Comment:
data-testid not possible because CKEditor field?

Reply:
yes!